### PR TITLE
say clockwise and counterclockwise in rot docstrings

### PR DESCRIPTION
**Pull request by oxinabox**

### CODE REVIEW

File: base/arraymath.jl [hunk 1]
```diff
@@ -244,7 +244,7 @@ end
 """
     rotr90(A, k)
 
-Rotate matrix `A` right 90 degrees an integer `k` number of times. If `k` is zero or a
+Rotate matrix `A` right 90 degrees clock-wise an integer `k` number of times. If `k` is zero or a
```
Comment (kshyatt) [inline base/arraymath.jl]: Since we say left above, should we say right here?

Comment (oxinabox) [reply base/arraymath.jl]: Or should we go the other way and not mention Left in the other one.
Eitehr way this  should  not say  `right 90 degrees clock-wise`

File: base/arraymath.jl [hunk 2]
```diff
@@ -204,7 +204,7 @@ end
 """
     rotl90(A, k)
 
-Rotate matrix `A` left 90 degrees an integer `k` number of times.
+Left rotate matrix `A` 90 degrees counter-clockwise an integer `k` number of times.
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
Rotate matrix `A` 90 degrees counterclockwise an integer `k` number of times.
```
Counterclockwise is one word.

File: base/arraymath.jl (hunk 1 above)
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
Rotate matrix `A` right 90 degrees clockwise an integer `k` number of times. If `k` is zero or a
```
Likewise, "clockwise" is a single word.

**Review by StefanKarpinski — CHANGES_REQUESTED** [review]

File: base/arraymath.jl [hunk 3]
```diff
@@ -244,7 +244,7 @@ end
 """
     rotr90(A, k)
 
-Rotate matrix `A` right 90 degrees an integer `k` number of times. If `k` is zero or a
+Rotate matrix `A` right 90 degrees clockwise an integer `k` number of times. If `k` is zero or a
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
Rotate matrix `A` 90 degrees clockwise an integer `k` number of times. If `k` is zero or a
```

**Review by StefanKarpinski — CHANGES_REQUESTED** [review]

**Comment by oxinabox** [comment]
I do  worry now that there is no  reference  to left or right in the docstrings
as  to if the `r` and `l` in the function name is more confusing

File: base/arraymath.jl [hunk 4]
```diff
@@ -204,7 +204,7 @@ end
 """
     rotl90(A, k)
 
-Rotate matrix `A` left 90 degrees an integer `k` number of times.
+Rotate matrix `A` 90 degrees counterclockwise an integer `k` number of times.
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
Left-rotate matrix `A` 90 degrees counterclockwise an integer `k` number of times.
```

File: base/arraymath.jl [hunk 5]
```diff
@@ -244,7 +244,7 @@ end
 """
     rotr90(A, k)
 
-Rotate matrix `A` right 90 degrees an integer `k` number of times. If `k` is zero or a
+Rotate matrix `A` 90 degrees clockwise an integer `k` number of times. If `k` is zero or a
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
Right-rotate matrix `A` 90 degrees clockwise an integer `k` number of times.
```

File: base/arraymath.jl [hunk 6]
```diff
@@ -244,7 +244,7 @@ end
...
 multiple of four, this is equivalent to a `copy`.
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
If `k` is a multiple of four (including zero), this is equivalent to a `copy`.
```

File: base/arraymath.jl [hunk 7]
```diff
@@ -204,7 +204,7 @@ end
 """
     rotl90(A, k)
 
-Rotate matrix `A` left 90 degrees an integer `k` number of times.
+Rotate matrix `A` 90 degrees counterclockwise an integer `k` number of times.
 If `k` is zero or a multiple of four, this is equivalent to a `copy`.
```
Comment (StefanKarpinski) [inline base/arraymath.jl]:
```suggestion
If `k` is a multiple of four (including zero), this is equivalent to a `copy`.
```

**Review by StefanKarpinski — DISMISSED** [review]

**Comment by StefanKarpinski** [comment]
> I do worry now that there is no reference to left or right in the docstrings
> as to if the `r` and `l` in the function name is more confusing

I think that's a valid concern. Have made corresponding suggestions.

**Comment by oxinabox** [comment]
I'm happy with this

**Pull request merged**